--- /usr/share/php/VersionControl/SVN/Checkout.php	2011-03-28 18:03:39.000000000 +0200
+++ Checkout.php	2011-11-09 10:37:01.000000000 +0100
@@ -157,6 +157,7 @@
                                 'ignore-externals',
                                 'no-auth-cache',
                                 'no_auth_cache',
+                                'trust-server-cert',
                                 'non-interactive',
                                 'non_interactive',
                                 'config-dir',
@@ -238,6 +239,7 @@
                     case 'non-recursive':
                     case 'non-interactive':
                     case 'no-auth-cache':
+                    case 'trust-server-cert':
                     case 'quiet':
                         if ($val === true) {
                             $_switches .= "--$switch ";
